meson.build

```diff
@@ -23,10 +23,12 @@ endif
 
 cpu = host_machine.cpu_family()
 
-if cpu == 'x86_64'
```

Comment:
I'm not quite sure what the benefit of doing this is. The default SIMD extension for x86_64 is sse2 which is very conservative and an ARM chip that predates neon is going to struggle running VNC anyhow. SIMD on aarch64 is part of the architecture and does not require an extension.

In any case, if you really wish to turn off auto-vectorization, you need to compile with -O2 or lower. This can be achieved by not using `--buildtype=release`.

Reply:
Is adding an extra option problematic?
Can change the default to `true` instead so that its not going to change for others not on gentoo.

Reply:
Well, honestly, I don't like the structure. This is also because I'm not overly fond of the solution that I have in the first place. Perhaps it's better to define two options instead: x86_64-cflags and arm-cflags and set the default so that SIMD modules are enabled.

Reply:
Any of the two is fine with me, as long as there is a toggle.
I don't want to enforce cpu flags as that is one of the more dangerous things to do, example: in the case of a cross-build environment for largescale deployment.

Reply:
Ok, let's just remove the SIMD flags and let the user set all CPU specific CFLAGS if they so desire. There isn't much benefit anyway with the default option of sse2 and this is the second time that it gets in the way of a package maintainer.